test/it/DissociateUcrSpec.scala

```diff
@@ -46,11 +48,9 @@ class DissociateUcrSpec extends IntegrationSpec {
         val response = post(controllers.consolidations.routes.DisassociateUcrController.submit(), "kind" -> "mucr", "mucr" -> "GB/321-54321")
 
         // Then
-        status(response) mustBe SEE_OTHER
-        redirectLocation(response) mustBe Some(controllers.consolidations.routes.DisassociateUcrSummaryController.displayPage().url)
-        theAnswersFor("eori") mustBe Some(
-          DisassociateUcrAnswers(ucr = Some(DisassociateUcr(kind = DisassociateKind.Mucr, mucr = Some("GB/321-54321"), ducr = None)))
-        )
+        intercept[RuntimeException] {
+          status(response)
```

Comment:
Where is an assertion? Is it on purpose?

Reply:
No, it's a mistake.  Addressing with Patryk's comment